src/WinRPM.jl

```diff
@@ -486,7 +486,7 @@ function do_install(package::Package)
                     end
                 end
             end
-            isfile(cpio) && rm(cpio)
+            (try; isfile(cpio); catch; false end;) && rm(cpio)
```

Comment:
when would this throw?

Reply:
Good question, it did though... I think this is the whole `isfile` can throw on windows issue...Maybe it got fixed on latest master. @kristofferc, you're probably up to date on this?

Reply:
with what input path?

Reply:
`isfile` throws on Windows for url for example, see https://github.com/JuliaLang/julia/issues/26685

Reply:
when is `cpio` ever going to have colons? it shouldn't include `http://` or things like that

Reply:
I was just saying when `isfile` would fail. Whether this ever can happen here I don't know.